frontend/.eslintrc.cjs

```diff
@@ -12,4 +12,7 @@ module.exports = {
     parserOptions: {
         ecmaVersion: 'latest',
     },
+    rules: {
+        'vue/multi-word-component-names': ['warn'],
```

Comment:
What is this for ?

Reply:
See https://eslint.vuejs.org/rules/multi-word-component-names.html

In vue, you are not allowed to write component with single name like `Main`, `Navbar`. But `QuizzCard` is okay since it's pascal case and can be turned into `quizz-card`.

This behavior is the default and is to prevent to have components with the same name as the html native components